Octokit.Tests/OctoKit.Tests-NetCore45.csproj

```diff
@@ -167,6 +167,10 @@
       <Project>{c8bc13b6-3fa3-4716-827d-e7706f976fe1}</Project>
       <Name>Octokit-NetCore45</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Octokit\Octokit.csproj">
```

Comment:
I'm not sure why these references are necessary.

Could you :fire: this one and the one in `Octokit.Tests-Portable`?